presto-raptor/src/main/java/com/facebook/presto/raptor/storage/StorageManagerConfig.java

```diff
@@ -188,18 +205,18 @@ public StorageManagerConfig setRecoveryThreads(int recoveryThreads)
         return this;
     }
 
-    @Config("storage.max-compaction-threads")
```

Comment:
Use `@LegacyConfig` for the old name and pull this rename into a separate commit